src/app/components/trade/LiquidityDetails.tsx

```diff
@@ -219,19 +220,25 @@ const PoolRecord = ({
   const { share, reward } = getShareReward(pair, balance, totalReward);
 
   return (
-    <ListItem border={border}>
-      <DataText>{`${aBalance?.currency.symbol || '...'} / ${bBalance?.currency.symbol || '...'}`}</DataText>
-      <DataText>
-        {`${aBalance.toFixed(2, { groupSeparator: ',' }) || '...'} ${aBalance?.currency.symbol || '...'}`}
-        <br />
-        {`${bBalance.toFixed(2, { groupSeparator: ',' }) || '...'} ${bBalance?.currency.symbol || '...'}`}
-      </DataText>
-      {upSmall && <DataText>{`${share.multiply(100).toFixed(4) || '---'}%`}</DataText>}
-      {upSmall && <DataText>{`~ ${reward.toFixed(4, { groupSeparator: ',' }) || '---'} BALN`}</DataText>}
-      <DataText>
-        <WithdrawText pair={pair} balance={balance} poolId={poolId} />
-      </DataText>
-    </ListItem>
+    <>
+      {parseFloat(share.multiply(100).toFixed(4)) > LOWEST_LIQUIDITY_POOL_PERCENT ? (
```

Comment:
Can we use comparison methods of `Fraction` directly? @shallcdn

Reply:
`share.multiply(100).greaterThan(LOWEST_LIQUIDITY_POOL_PERCENT_FRAC)`
and
`LOWEST_LIQUIDITY_POOL_PERCENT_FRAC = new Fraction(1, 100)`

Reply:
@lijac  Yes, we can, but I wonder why don't we use pure JS as the way I wrote the codes? Do we have any reasons? 
Ext: easy to read, consistency... I just want to know the reason to prevent my mistakes for any PRs in the future.

Reply:
Sure, here are the reasons I can think of:
1. We may need to update `Fraction` class in the future and that can be `toFixed` method. If that's the case, we will probably update `greaterThan` to accommodate new changes but cannot guarantee `toFixed(4)` still returns the desired value.
2. We also have `BigNumber` in the project for processing numbers (though it is going to fade out), it's better to use explicit functions of separate classes to prevent confusion.

Personal preference also has a place in my mind :)

Reply:
@lijac I have updated, Please help take a look again, thank you!